Src/CSharpier.Tests/TestFiles/FieldDeclaration/FieldDeclarations.cst

```diff
@@ -30,4 +30,29 @@ public class ClassName
         Property4 = false,
         Property5 = false
     };
+
+    // leading comments
+    private static const string SomeName = "1";
+
+    public SomeClass SomeName; // trailing comments
+
+    public System.Text.StringBuilder NamespacedField;
+
+    public static Dictionary<Type, string> PropertiesByType = new Dictionary<
```

Comment:
I think it makes sense to break on the assignment first, i.e.
```
public static Dictionary<Type, string> PropertiesByType =
    new Dictionary<Type, string>();
```

In fact, I'm a little confused why this isn't happening already. The assignment expression is the outermost `Group()` right? It should break on that before the its children (which presumably are the left and right sides)

Reply:
There were a few edge cases that popped up due to the changes I was making. I created issues for some of them because this PR was already getting large.
Getting this to break after the = caused a lot of other cases to change. I think this one requires conditional groups (which don't exist yet), or possibly an ifBreak. 
https://github.com/belav/csharpier/issues/239